config/admin.js

```diff
@@ -1,4 +1,7 @@
 module.exports = ({ env }) => ({
+  apiToken: {
+    salt: env('API_TOKEN_SALT','b4dbad35-6caa-4f84-be56-90d0d3bc2fdf')
```

Comment:
Unless I'm misunderstanding `apiToken.salt` and `auth.secret`, I think that these should probably have different default values in the case where the `API_TOKEN_SALT` and `ADMIN_JWT_SECRET` environment variables are not defined.

Reply:
Hi @zach-render thanks for the comment! In production the secrets will be generated by the render.yaml file: https://github.com/render-examples/strapi-sqlite/pull/24/commits/a17579f17c9f14d201c9aa4546ee42d3d2e44e3a 
And in development it does not matter, because things are only local and ephemeral anyway. 
But you can also change the value before merging. 😀👍